fuel-client/assets/schema.sdl

```diff
@@ -131,6 +131,7 @@ enum CoinStatus {
 type Contract {
 	id: ContractId!
 	bytecode: HexString!
+	salt: Salt!
```

Comment:
I originally tried doing it this way, but I found args are only valid on queries, not plain Graphql types. Would there be a way to do it like this? When I talked with @Voxelot he suggested making a single specific balance query and breaking away balances from the default contract object (in addition to a balances query which I am planning on covering in the next PR)

Reply:
I'd have to take a closer look to see if async-graphql / cynic support this type of query structure @AlicanC. I think deferring balances to a separate PR is a good idea.

Reply:
> I originally tried doing it this way, but I found args are only valid on queries, not plain Graphql types.

In GraphQL speak, `contractBalance` is a field of the type `Query` and it has arguments. `Contract` is yet another type like `Query` and it could as well have a similar field.

I thought there was an example of this in the repo but couldn't find any. I guess we'll leave it to whoever takes the follow-up issue :D